src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -733,15 +733,37 @@ public void analyse(Network network, List<PropagatedContingency> contingencies,
         LOGGER.info("Running DC sensitivity analysis with {} factors and {} contingencies",  lfFactors.size(), contingencies.size());
 
         // create DC load flow engine for setting the function reference
-        DcLoadFlowParameters dcLoadFlowParameters = new DcLoadFlowParameters(slackBusSelector, matrixFactory,
-            true, lfParameters.isDcUseTransformerRatio(), lfParameters.isDistributedSlack(), lfParameters.getBalanceType(), true,
-            lfParametersExt.getPlausibleActivePowerLimit(), lfParametersExt.isAddRatioToLinesWithDifferentNominalVoltageAtBothEnds(), true, true, lfParameters.getCountriesToBalance());
+        var networkParameters = new LfNetworkParameters(slackBusSelector,
```

Comment:
It would be a bit clearer if you put those lines apart in a `createDcLoadFlowParameters(lfParameters, lfParametersExt, slackBusSelector)` method

Reply:
Done